src/main/drivers/accgyro/accgyro_spi_icm426xx.c

```diff
@@ -193,6 +201,24 @@ bool icm426xxSpiAccDetect(accDev_t *acc)
 
 static aafConfig_t getGyroAafConfig(void);
 
+static void turnGyroAccOff(const extDevice_t *dev)
+{
+    spiWriteReg(dev, ICM426XX_RA_PWR_MGMT0, ICM426XX_PWR_MGMT0_GYRO_ACCEL_MODE_OFF);
+    delay(15);
```

Comment:
What is the reason for the delay?

Reply:
This one was just a bit of extra caution in case off some undocumented weirdness going on after turning the gyro and acc off. 
I'll look into getting rid of it now that we know that it works.

Reply:
this delay should be removed now